packages/codegen/src/types-builder.ts

```diff
@@ -37,6 +37,13 @@ export const defaultDeclarations = (): CodeDeclarations => ({
   takenNames: new Set(),
 })
 
+const opaqueHashers = new Set<string>([
+  "Blake2128",
+  "Blake2256",
+  "Twox128",
+  "Twox256",
+])
```

Comment:
If all the non-opaque hashers end with "Concat", wouldn't checking for this condition be a safer?

Otherwise it sounds like we will have to keep track of which hashers are being added to maintain this list.

Edit - No, `Identity` is non-opaque and doesn't have Concat in it. Yep, a Set is better then 👍